src/Creative/InLine/Linear/MediaFile.php

```diff
@@ -70,10 +70,22 @@ public function setUrl($url)
 
     /**
      * @param int $bitrate
+     * @return $this
      */
     public function setBitrate($bitrate)
     {
         $this->domElement->setAttribute('bitrate', (int) $bitrate);
         return $this;
     }
+
+    /**
+     * @param string $value
+     * @return $this
+     */
+    public function setApiFramework($value)
+    {
+        $this->domElement->setAttribute('apiFramework', (string) $value);
```

Comment:
This feature is deprecated.

```
identifies the API needed to execute an interactive media file, but current
support is for backward compatibility. Please use the
<InteractiveCreativeFile> element to include files that require an API
for execution.
```

Add comment about that in PHPDoc.

Reply:
Missed that from IAB docs. Will review them again.

Reply:
Just checked [VPAID2 docs](https://www.iab.com/wp-content/uploads/2015/06/VPAID_2_0_Final_04-10-2012.pdf). According to `2.5.1 API Framework`:

```
When a VPAID ad unit is referenced from a VAST document, the value for the
apiFramework attribute in the <MediaFile> element must be VPAID (all caps). This
attribute identifies the VPAID API for the creative. Version information should be handled by the
VPAID handshakeVersion() call (rather than identified in the VAST file). See Section
3.1.1 for more information.
```

Reply:
I don't mind about this attribute. This library try to support all versions of VAST protocol, as some servers support different and even not documenmted attributes, so any feature, if it is deprecated, must be documented as deprecated from concrete version, or must be some description about its useage. Code must be documentation.

```
/**
 * This feature is deprecated since v. 1.2.3, please use "blahblah" instead
```

Quote from VAST 4.1: https://iabtechlab.com/wp-content/uploads/2018/06/VAST4.1-June7-FINAL.pdf
> Deprecating Video Player Ad-Serving Interface Definition (VPAID): While VPAID
will likely be in use for some more time, with VAST 4.1 we are taking the first steps
to officially deprecate the use of VPAID. The apiFramework attribute on MediaFile,
and the conditionalAd attribute on the Ad element are being deprecated.

Reply:
O. Haven't looked at VAST4 yet. It's a reason to party if partner works with VAST3 :)
I mean, does really somebody uses VAST4 already and is there any player that supports v4?

But you're right. Even if nobody uses it, it still makes sense to at least add comment about deprecation. I will make it just comment, not `@deprecated`, correct?

Reply:
Yes, just description. Feature deprecated in protocol, not in library

Reply:
Already done, then.